Documentation/contributing/release/backports.rst

```diff
@@ -209,3 +216,22 @@ listed in the pull request description.
 .. code-block:: bash
 
    $ for pr in 12589 12568; do contrib/backporting/set-labels.py $pr done 1.8; done
+
+Backporting guide for the original committer
+--------------------------------------------
+
+Mark your PRs needing backport as ``needs-backport/X.Y``.  At some point, your
+changes will be picked up on a backport PR and you will be notified and asked to
+approve the backport commits.  Please check:
+
+#. That all the commits from your PR have been indeed backported.
+#. In case of conflicts, the resulting changes look good.
+
+
+Backporting guide for the merger
+--------------------------------
+
+When merging a backport PR, please set the labels of the backported PRs to
+``done``. Typically, backport PRs include a line on how do that. E.g.,: ``for pr
+in 12894 12621 12973 12977 12952; do contrib/backporting/set-labels.py $pr done
+1.8; done``
```

Comment:
This needs to have a tag `upstream-prs`, as in:

     ```upstream-prs
     for pr in 12894 12621 12973 12977 12952; do contrib/backporting/set-labels.py $pr done 1.8; done
     ```

Could detail that? I had it on my TODO list to document for a while but didn't get to it.

Reply:
The visual result of the above was a bit strange:

![image](https://user-images.githubusercontent.com/221481/93332574-5103c700-f822-11ea-94a6-661a3bf88206.png)

So I tried this instead:

```
.. code-block:: bash

   $ for pr in 12894 12621 12973 12977 12952; do contrib/backporting/set-labels.py $pr done 1.8; done
```

Which looks better, but I'm not sure how to add the tag.

Reply:
Ah, I wasn't clear. What I mean if that we should also document the `upstream-prs` part, which is present in backport PRs. For instance, you can see it if you edit https://github.com/cilium/cilium/pull/13184.

In rst, you may need something like:
```rst
.. :

    ```upstream-prs
    for pr in 12894 12621 12973 12977 12952; do contrib/backporting/set-labels.py $pr done 1.8; done
    ```
```

Reply:
Added an additional patch to document the importance of `upstream-prs`